Misc/NEWS

```diff
@@ -10,6 +10,10 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30876: Relative import from unloaded package now reimports the package
```

Comment:
I first uninstalled blurb. If install blurb, the `--upgrade` option doesn't have any effect.
```
$ python3 -m pip install --user --upgrade blurb
Requirement already up-to-date: blurb in /home/serhiy/.local/lib/python3.5/site-packages
```